### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [jarellano01/template_jekyll-tutorials#1]
The user @carlsonsantana validate your site template "https://jarellano01.github.io/template_jekyll-tutorials/" and found these accessibility errors:
* Page https://jarellano01.github.io/template_jekyll-tutorials/:
  * Issue *c0213e9a-965e-48dd-8e2a-7cbb682c84f3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.49:1. Recommendation: change background to #2d363e.
    * **Selector**: `html > body > nav > div > div:nth-child(2) > ul > li:nth-child(3) > a`
    * **Context**: `<a href="/template_jekyll-tutorials/blog/">Blog</a>`
  * Issue *ace6649e-9b5e-44be-ad1d-c675573a99bd*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.49:1. Recommendation: change background to #2d363e.
    * **Selector**: `html > body > nav > div > div:nth-child(2) > ul > li:nth-child(2) > a`
    * **Context**: `<a href="/template_jekyll-tutorials/tutorials/">Tutorials</a>`
  * Issue *c3a5207d-3fc3-4be9-b38c-c7902fa948c6*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.49:1. Recommendation: change background to #2d363e.
    * **Selector**: `html > body > nav > div > div:nth-child(2) > ul > li:nth-child(1) > a`
    * **Context**: `<a href="/template_jekyll-tutorials/">Home</a>`
  * Issue *2f9565c6-aa03-41bf-bbc1-2eb59d543491*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
    * **Message**: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
    * **Selector**: `html > body > nav > div > div:nth-child(1) > a > img`
    * **Context**: `<img class="img-responsive2" src="/template_jekyll-tutorials/assets/images/ja-logo.png">`
  * Issue *79e5d5a2-5bf9-47e9-90a2-7b4983d86b0d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.49:1. Recommendation: change background to #2d363e.
    * **Selector**: `html > body > nav > div > div:nth-child(1) > a`
    * **Context**: `<a class="navbar-brand" href="/template_jekyll-tutorials/"><img class="img-responsive2" sr...</a>`
  * Issue *99afbbb2-e02b-4095-9e02-fa39a9be63a5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
    <meta charset="utf-8...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/4a59f854-47e1-4ce2-9e94-2ada9ee4a2bb/).
